### PR TITLE
Arrangement Works

### PR DESCRIPTION
**Pull request by onting**
Write license acknowledgement.
Adjust file locations.

### CODE REVIEW

File: ibus/LICENSE [hunk 1]
```diff
@@ -0,0 +1,350 @@
+This source code on this software based on ibus-hangul engine code.
+We gratefully acknowledge these and other contributions to libhanjp.
+We recommend that programs that use libhanjp also acknowledge these contributions,
+and quote these license statements in an appendix to the documentation.
+
+ibus-hangul
+https://github.com/phuang/ibus-hangul
+GNU General Public License v2.0
+Huang Peng <shawn.p.huang@gmail.com>
+Choe Hwanjin <choe.hwanjin@gmail.com>
```
Comment (sukso96100) [inline ibus/LICENSE:10]: Rather then putting acknowledgement on `LICENSE` file, consider putting this on `README.md`. As `README.md` is more easily found by other visitors.
AFAIK, `LICENSE` usually only contains full license text. And other thing like acknowledgements are usually included in `README.md` or other separate file such as `THANKS` file.

**Comment by sukso96100** [comment]
**"Write license acknowledgement." and **"Adjust file locations."** are two separate different tasks. Next time, consider submitting separate Pull Request.(One PR for One Issue(Task)) That will be better and easier for both submitter and reviewer to review and discuss about PR.

File: ibus/LICENSE (hunk 1 above)
Comment (onting) [reply ibus/LICENSE:10]: Right, I've got missed.
I'll fix for it.

**Comment by onting** [comment]
Although that is good convention, we are still not in stable.
I reckon some mixing is required for fast work.
I'll try to do so, but please understand if there will be some mixing.

**Review by sukso96100 — APPROVED** [review]

**Pull request merged**